opentelemetry-collector/values.yaml

```diff
@@ -83,6 +83,33 @@ service:
   type: ClusterIP
   annotations: {}
 
+ports: # configures both service, and deployment
+  otlp:
+    enabled: true
+    containerPort: 55681 # new port could also be 4317
+    servicePort: 55681
+    protocol: TCP
+  otlp-http:
+    enabled: true
+    containerPort: 55681 # http server starts up on 55681
+    servicePort: 55681
+    protocol: TCP
+  jaeger-thrift:
+    enabled: true
+    containerPort: 14268
+    servicePort: 14268
+    protocol: TCP
+  jaeger-grpc:
+    enabled: true
+    containerPort: 14250
+    servicePort: 14250
+    protocol: TCP
+  zipkin:
```

Comment:
This may also be an opportunity to add the other not so popular Jaeger ports/formats for `thrift_binary` and `thrift_compact` which are ports: 6832 and 6831 respectively.
https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/jaegerreceiver/README.md

Reply:
mentioned this above, but the more common pattern (from my experience) is to put stock values / examples in the `values` file of the chart but make it an array or method that could be extended. This PR for my purposes is such that I'm creating a values entry in a private repo and pinning this chart to this uptream, where I can add name / port mappings that can override and/or extend the defaults provided

Reply:
What we want to make sure is all the default ports that the collector will listen on for OTLP, Jaeger, and Zipkin are enabled by default when someone does a simple install using `helm install ... --set ...` (no values file supplied)

The collector has long been an an odd state, where these 3 receivers are enabled by default but not all ports to use them are opened for it.